evaluation/tests/interface_tests/test-IFS.sh

```diff
@@ -0,0 +1,5 @@
+IFS=/
+curr_dir=/home/bolun/programming/pash
```

Comment:
We could "deanonymize" this by making this string to be `/test1/test2/test3/test4`?